napari/_vispy/canvas.py

```diff
@@ -634,6 +639,33 @@ def _add_overlay_to_visual(self, overlay: Overlay) -> None:
             vispy_overlay.node.parent = self.view.scene
         self._overlay_to_visual[overlay] = vispy_overlay
 
+    def _calculate_view_direction(self, event_pos) -> List[float]:
```

Comment:
Nice work! Makes sense to me :) For readability, would be nice to have more explicit variable names instead of relying on comments, like:

`p0` -> `pos_scene`
`p1` -> `eye_scene`

and so on :)